Description: Install FindLibSolv.cmake into ${libdir}/cmake/libsolv/.
Author: Mihai Moldovan <ionic@ionic.de>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -73,7 +73,7 @@ ENDIF (NOT PKGCONFIG_INSTALL_DIR)
 
 # where to look first for cmake modules, before ${CMAKE_ROOT}/Modules/ is checked
 SET (CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
-INSTALL( FILES ${CMAKE_MODULE_PATH}/FindLibSolv.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/share/cmake/Modules )
+INSTALL( FILES ${CMAKE_MODULE_PATH}/FindLibSolv.cmake DESTINATION ${LIB_INSTALL_DIR}/cmake/libsolv/ )
 
 INCLUDE (${CMAKE_SOURCE_DIR}/VERSION.cmake)
 
