Description: Install FindLibSolv.cmake into ${libdir}/cmake/libsolv/.
Author: Mihai Moldovan <ionic@ionic.de>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@
 
 # where to look first for cmake modules, before ${CMAKE_ROOT}/Modules/ is checked
 SET (CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
-INSTALL( FILES ${CMAKE_MODULE_PATH}/FindLibSolv.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/share/cmake/Modules )
+INSTALL( FILES ${CMAKE_MODULE_PATH}/FindLibSolv.cmake DESTINATION ${LIB_INSTALL_DIR}/cmake/libsolv/ )
 
 # for shared libraries on windows (DLLs), we just export all symbols for now
 SET(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
